src/data/logbook.ts

```diff
@@ -216,7 +216,6 @@ export const getLogbookMessage = (
         case "cold":
         case "gas":
         case "heat":
-        case "colightld":
```

Comment:
Intentional change or cruft on the branch?

Reply:
Ah, forgot to add a comment: Intentional change. This seems to be a very old legacy entry, if even Paulus is not aware of it: https://discord.com/channels/330944238910963714/674164904298676225/806654268378841128

That value is not used anywhere in core or frontend.